chains/evm/contracts/test/onRamp/OnRampOverSuperchainInterop/OnRampOverSuperchainInterop.hashInteropMessage.t.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.24;
+
+import {Internal} from "../../../libraries/Internal.sol";
+import {SuperchainInterop} from "../../../libraries/SuperchainInterop.sol";
+import {OnRampOverSuperchainInteropSetup} from "./OnRampOverSuperchainInteropSetup.t.sol";
+
+contract OnRampOverSuperchainInterop_hashInteropMessage is OnRampOverSuperchainInteropSetup {
+  function test_BasicMessageHash() public {
+    Internal.Any2EVMRampMessage memory message = _generateBasicAny2EVMMessage();
+
+    bytes32 messageHash = SuperchainInterop._hashInteropMessage(message, address(s_onRampOverSuperchainInterop));
+    bytes32 expectedMessageHash = Internal._hash(message, _getOffRampMetadataHash());
+
+    assertEq(messageHash, expectedMessageHash);
+  }
+
+  function test_TokenMessageHash() public {
```

Comment:
Again, not sure we need separate tests for something that is effectively the same. Can combine